packages/terra-table/tests/nightwatch/single-select-table/single-select-table-spec.js

```diff
@@ -1,119 +0,0 @@
-/* eslint-disable no-unused-expressions */
```

Comment:
Why did we drop these?

Reply:
because we want don't want unused expressions?

Reply:
not dropped, just moved to packages/terra-table/nightwatch/single-select-table-spec.js